crates/rnote-engine/src/pens/eraser.rs

```diff
@@ -20,15 +20,43 @@ pub enum EraserState {
     Down(Element),
 }
 
+#[derive(Debug, Default, Clone, Copy)]
+pub struct EraserMotion {
+    last_element: Option<(Element, Instant)>,
+    pub speed: f64,
+}
+
+impl EraserMotion {
+    pub const SMOOTHING_FACTOR: f64 = 3.0;
+
+    fn update(&mut self, element: Element) {
+        if let Some((last_element, last_element_time)) = self.last_element {
+            let delta = element.pos - last_element.pos;
+            let delta_time = Instant::now() - last_element_time;
```

Comment:
Actually this is not the correct way to do this. You shouldn't use `Instant::now` here. 
The reason is that on some devices, the pen refreshes faster than the screen rate and gtk4 sends all the events that happened between two time frames once and regroups them. Hence you can get more than one update with basically the same `Instant::now` but that happened at different time steps.

You have to propagate to `update` the Instant the pen event actually happens.

See the `_now` argument from here : 
https://github.com/flxzt/rnote/blob/0964cb5d34077a5d36346f6a4b78b90b5cdd5533/crates/rnote-engine/src/pens/eraser.rs#L84
Remove the `_` (it's going to be used) and propage the `now` variable to the `update` function. This is the one you're going to use instead of the `Instant::now`.

Reply:
This might be why I had issues. Fix this part and I will retest whether or not the issue persist on my side

Reply:
Should be fixed now, thanks for catching that!